{{ cookiecutter.project_slug }}/{{ cookiecutter.project_slug }}/users/views.py

```diff
@@ -36,8 +37,13 @@ def change_email_request(self, request, *args, **kwargs):
 
         user = request.user
         self.check_object_permissions(request, user)
-
-        user.new_email = serializer.data['email']
+        
+        new_email = serializer.data['email']
+                                    
+        if len(User.objects.filter(email=new_email)) > 0:
```

Comment:
Would probably make this:

```python
if User.objects.filter(email=new_email).first():
    raise ValidationError({'email': ['user with this email already exists.']})
```

for just a tad more idiomatic python, and a slightly better SQL query.